patches/13-kubernetes-core-collection-bump.patch

```diff
@@ -1,379 +0,0 @@
-index f43103b78..0fbae2455 100644
---- internal/generate/clusterserviceversion/clusterserviceversion_updaters.go
-+++ internal/generate/clusterserviceversion/clusterserviceversion_updaters.go
-@@ -425,7 +425,6 @@ type serviceportPath struct {
- // crdGroups groups the crds with similar service port and name. It returns a map of serviceportPath
- // and the corresponding crd names.
- func crdGroups(crdToConfig map[string]apiextv1.WebhookConversion) map[serviceportPath][]string {
--
- 	uniqueConfig := make(map[serviceportPath][]string)
- 
- 	for crdName, config := range crdToConfig {
-@@ -574,7 +573,7 @@ func mutatingToWebhookDescription(webhook admissionregv1.MutatingWebhook, depNam
- func findMatchingDeploymentAndServiceForWebhook(c *collector.Manifests, wcc admissionregv1.WebhookClientConfig) (depName string, ws *corev1.Service) {
- 	// Return if a service reference is not specified, since a URL will be in that case.
- 	if wcc.Service == nil {
--		return
-+		return depName, ws
- 	}
- 
- 	// Find the matching service, if any. The webhook server may be externally managed
-@@ -586,18 +585,18 @@ func findMatchingDeploymentAndServiceForWebhook(c *collector.Manifests, wcc admi
- 		}
- 	}
- 	if ws == nil {
--		return
-+		return depName, ws
- 	}
- 
- 	// Only ExternalName-type services cannot have selectors.
- 	if ws.Spec.Type == corev1.ServiceTypeExternalName {
--		return
-+		return depName, ws
- 	}
- 
- 	// If a selector does not exist, there is either an Endpoint or EndpointSlice object accompanying
- 	// the service so it should not be added to the CSV.
- 	if len(ws.Spec.Selector) == 0 {
--		return
-+		return depName, ws
- 	}
- 
- 	depName = findMatchingDepNameFromService(c, ws)
-diff --git a/internal/helm/client/secrets_watch.go b/internal/helm/client/secrets_watch.go
-index e9cd46801..a61c7532a 100644
---- internal/helm/client/secrets_watch.go
-+++ internal/helm/client/secrets_watch.go
-@@ -117,7 +117,7 @@ func (w *WatchedSecrets) Watch(ctx context.Context, opts metav1.ListOptions) (wa
- 	return w.inner.Watch(ctx, opts)
- }
- 
--func (w *WatchedSecrets) Patch(ctx context.Context, name string, pt types.PatchType, data []byte, opts metav1.PatchOptions, subresources ...string) (result *corev1.Secret, err error) {
-+func (w *WatchedSecrets) Patch(ctx context.Context, name string, pt types.PatchType, data []byte, opts metav1.PatchOptions, _ ...string) (result *corev1.Secret, err error) {
- 	return w.inner.Patch(ctx, name, pt, data, opts)
- }
- 
-diff --git a/internal/helm/release/manager.go b/internal/helm/release/manager.go
-index 07641ccc7..72a246461 100644
---- internal/helm/release/manager.go
-+++ internal/helm/release/manager.go
-@@ -95,7 +95,7 @@ func (m manager) IsUpgradeRequired() bool {
- 
- // Sync ensures the Helm storage backend is in sync with the status of the
- // custom resource.
--func (m *manager) Sync(ctx context.Context) error {
-+func (m *manager) Sync(_ context.Context) error {
```

Comment:
There are non-ansible changes in the patch. Are these still required and added as part of another patch file?

Reply:
This patch file is not required as most of the changes in the patch are already added in the upstream code. Secondly, these patches were also not related to the kubernetes.core bump. As we have moved the code related to ansible-operator to ansible-operator-plugins, these patch file is no longer required. I have removed the file in  https://github.com/openshift/ocp-release-operator-sdk/pull/387/commits/6aae506ab65bb7aab8e07662166ec9685367cd6d.